core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1162,7 +1162,7 @@ private[spark] class DAGScheduler(
         partitions = stage.rdd.partitions
       }
 
-      if (taskBinaryBytes.length * 1000 > TaskSetManager.TASK_SIZE_TO_WARN_KB) {
+      if (taskBinaryBytes.length > TaskSetManager.TASK_SIZE_TO_WARN_KB * 1000) {
```

Comment:
More precise I think would be 1024, is it right?

Reply:
it's KB not KiB, isn't it?

Reply:
We use 1024 at L497:

```scala
if (serializedTask.limit() > TaskSetManager.TASK_SIZE_TO_WARN_KB * 1024 &&
  !emittedTaskSizeWarning) {
  emittedTaskSizeWarning = true
  logWarning(s"Stage ${task.stageId} contains a task of very large size " +
    s"(${serializedTask.limit() / 1024} KB). The maximum recommended task size is " +
    s"${TaskSetManager.TASK_SIZE_TO_WARN_KB} KB.")
}
```

Shall we use a consistent value?

Reply:
yea, I think 1024 is better..

Reply:
Sounds good; I'll fix the constant name while here.